### ISSUE TITLE
Creating a new alert/recall: Word "button" is missing after "Submit"

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#76]
URL: https://hcdev.openplus.ca/en/documentation/creating-new-alertrecall
User Agent: Chrome
Language: English

Issue Description: In the third bullet of "Create an alert" and "Create a recall", word "button" is missing after "Submit". "Submit" is a button, therefore, when referring to it, "button" should be identified.
![image](https://user-images.githubusercontent.com/88776404/129224546-2d9f175e-9176-42f0-b020-18bd1c6bf629.png)

Steps to reproduce: Visual check.
Expected results: The third bullet of "Create an alert" and "Create a recall" should be updated to "3. Click **Submit** button to create and save a draft of your new alert." "3. Click **Submit** button to create and save a draft of your new recall."

**Comment by echouin** [comment]
Corrected in EN (and in FR):
![image](https://user-images.githubusercontent.com/88099454/148241294-bfbf1f2d-92b7-4c89-8faf-e7908ff0b39b.png)

**Comment by hydegeoff** [comment]
Issue fixed
